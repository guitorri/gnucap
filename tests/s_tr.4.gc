spice
.option list
v1 (1 0) pulse (0 1)
r1 (2 0) 1
.print tran v(1)
.tran 0 1 .1
.tran
.end
