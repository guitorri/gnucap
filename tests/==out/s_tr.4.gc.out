"v1 (1 0) pulse (0 1) "
"r1 (2 0) 1 "
".print tran v(1) "
".tran 0 1 .1 "
#Time       v(1)      
 0.         0.        
 0.1        1.        
 0.2        1.        
 0.3        1.        
 0.4        1.        
 0.5        1.        
 0.6        1.        
 0.7        1.        
 0.8        1.        
 0.9        1.        
 1.         1.        
".tran "
#Time       v(1)      
 1.         1.        
 1.1        1.        
 1.2        1.        
 1.3        1.        
 1.4        1.        
 1.5        1.        
 1.6        1.        
 1.7        1.        
 1.8        1.        
 1.9        1.        
 2.         1.        
".end "
